### PR TITLE
Update Release notes

### PR DESCRIPTION
**Pull request by AnPucel**
## Description

Add changelog to release notes in csproj. Remove `No Copyright` note. 

## How has this been tested?

N/A

### CODE REVIEW

File: src/Vault/Vault.csproj [hunk 1]
```diff
@@ -10,19 +10,19 @@
     <Company>Hashicorp</Company>
     <AssemblyTitle>Vault OpenAPI Library</AssemblyTitle>
     <Description>A Vault Client library generated from an OpenAPI doc</Description>
-    <Copyright>No Copyright</Copyright>
```
Comment (dhuckins) [inline src/Vault/Vault.csproj:13]: we should update this over removing it
copyright headers got added in https://github.com/hashicorp/vault-client-dotnet/pull/96 (not sure what to put in this xml tho)

File: src/Vault/Vault.csproj [hunk 2]
```diff
@@ -10,19 +10,19 @@
...
     <RootNamespace>Vault</RootNamespace>
     <Version>0.1.0-beta</Version>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Vault.xml</DocumentationFile>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageLicenseExpression>MPL-2.0</PackageLicenseExpression>
     <RepositoryUrl>https://github.com/hashicorp/vault-client-dotnet.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <PackageReleaseNotes>Minor update</PackageReleaseNotes>
```
Comment (dhuckins) [inline src/Vault/Vault.csproj:21]: all the other xml names with file paths have the word "File" at the end of it (eg: "DocumentationFile", "PackageReadmeFile") does the build tool know to switch from raw strings vs file path?

File: src/Vault/Vault.csproj (hunk 1 above)
Comment (AnPucel) [reply src/Vault/Vault.csproj:13]: Yeah. I wanted to remove it for now b/c it shows up when you publish it to Nuget. I don't want it saying "No copyright" there. But agreed. we should update it.

File: src/Vault/Vault.csproj (hunk 2 above)
Comment (AnPucel) [reply src/Vault/Vault.csproj:21]: good call. Lemme double check this. I just packed the nuget file to test. But yeah, I'll see if let's you pipe in a release notes

File: src/Vault/Vault.csproj (hunk 1 above)
Comment (AnPucel) [reply src/Vault/Vault.csproj:13]: I looked at a few other Nuget packages under Hashicorp and didn't see a copy right there, but I'll see what we should change it to

**Review by averche — APPROVED** [review]

**Pull request merged**